packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts

```diff
@@ -143,6 +143,13 @@ export interface LoadBalancedServiceBaseProps {
    * @default CloudFormation-generated name
    */
   readonly serviceName?: string;
+
+  /**
+   * The LogDriver to use for logging.
+   *
+   * @default AwsLogDriver if enableLogging is true
```

Comment:
Use “-“ for the default value if it’s not a real value

Reply:
Updated